src/Website/Resources/ordercancelemail.php

```diff
@@ -1,21 +1,30 @@
 <?php
 require 'PHPMailerAutoload.php';
-
-$mail = new PHPMailer;
+require 'logindata.php';
 
 //$mail->SMTPDebug = 3;                               // Enable verbose debug output
 
+
+
+
+$mail = new PHPMailer(true);
+
+
+
+
+
+
 $mail->isSMTP();                                      // Set mailer to use SMTP
 $mail->Host = $mailHost;  // Specify main and backup SMTP servers
 $mail->SMTPAuth = true;                               // Enable SMTP authentication
 $mail->Username = $mailUser;                 // SMTP username // apikey CS3250Key2
 $mail->Password = $mailword;                           // SMTP password  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
 $mail->SMTPSecure = $mailSecure;                            // Enable TLS encryption, `ssl` also accepted
-$mail->Port = $mailPort;                                  // TCP port to connect to
+$mail->Port = $mailPort;                                    // TCP port to connect to
 
 $mail->setFrom('stuffbuysharks@gmail.com', 'Shark Mailer');
 $mail->addAddress($_SESSION["username"]);// Add a recipient    
-$mail->addCC($_SESSION["email"] );         //Add CC    // Name is optional
+//$mail->addCC($_SESSION["email"] );         //Add CC    // Name is optional
```

Comment:
are we no longer sending a cc to the email they entered?

Reply:
Not from the cancelation because there is no place to enter the second email at this time. Causes an error.